### PR TITLE
DEP Update dependency versions

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
I think we need to update the travis file as well https://github.com/silverstripe/cwp-recipe-search/blob/2.9/.travis.yml

**Comment by emteknetnz** [comment]
Updated

File: .travis.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ import:
 
 env:
   global:
-    - REQUIRE_RECIPE="2.8.x-dev"
+    - REQUIRE_RECIPE="2.9.x-dev"
```
Comment (maxime-rainville) [inline .travis.yml]: We don't have a cwp 2.9.x and never will. The fix would be to use something other than `silverstripe/silverstripe-travis-shared:config/provision/cwp-jobs-fixed.yml`

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml]: Updated to use 4.9.x-dev standard recipe instead

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
I think the build is failing because the the simple theme doesn't have a search template.

I think we either need to update this recipe to use the Watea theme for its build or to install the new sink.

**Comment by emteknetnz** [comment]
I've added the starter theme to match what was in the cwp provision https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/provision/cwp.yml#L6

**Comment by emteknetnz** [comment]
I've manually defined the jobs to exclude postgres which isn't compatible

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**